src/test/scala/com/cognite/spark/datasource/EventsRelationTest.scala

```diff
@@ -563,8 +566,8 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
         .option("onconflict", "update")
         .save()
     }
-    e.getCause shouldBe a[CdpApiException]
-    val cdpApiException = e.getCause.asInstanceOf[CdpApiException]
+    e.getCause shouldBe a[com.cognite.sdk.scala.common.CdpApiException]
```

Comment:
Makes more sense to have it imported and not need to namespace it like this, imo.

Reply:
Agreed, I'll rename the `CdpApiException` in `CdpConnector` to something else (we can replace it with the SdkException in another PR).

Reply:
Good point. This was to get around the name collision with `CdpConnector.CdpApiException`, but that could be replaced with the one from the sdk without too much hassle.